code/modules/power/lights.dm

```diff
@@ -164,17 +164,17 @@
   */
 /obj/machinery/light
 	name = "light fixture"
+	desc = "Yep, its a light."
```

Comment:
Technically it IS actually a fixture, not a light. The light is attached to it.

Reply:
☝️🤓